pkg/render/node.go

```diff
@@ -928,16 +929,13 @@ func (c *nodeComponent) nodeEnvVars() []v1.EnvVar {
 
 	// Determine MTU to use. If specified explicitly, use that. Otherwise, set defaults based on an overall
 	// MTU of 1460.
-	ipipMtu := "1440"
```

Comment:
I have some slight hesitation that we're going from some defaulted values to auto-detection, the concern being if some how the auto-detected values are not correct and causes things to break.
Since we defaulted to fairly small MTUs, most likely auto-detected value will be higher if anything. If those higher values are incorrect then once new pods are started could that cause traffic to not function as expected?

My concerns are mostly FUD, I don't have any concerns based on fact.

Reply:
Yeah, you're right that this might change MTUs, but that is sorta the intention of this feature :)